web-react-ts/package-lock.json

```diff
@@ -22173,8 +22173,7 @@
       "dev": true
     },
     "autoprefixer": {
-      "version": "10.4.5",
-      "resolved": "https://registry.npmjs.org/autoprefixer/-/autoprefixer-10.4.5.tgz",
```

Comment:
You deleted this line? Or it's an accident?

Reply:
I think when I run npm install after deleting my node modules